.projenrc.js

```diff
@@ -12,7 +12,7 @@ const project = new Cdk8sTeamJsiiProject({
   ],
 
   bundledDeps: [
-    'yaml@2.0.0-7',
+    'yaml@2.2.2',
```

Comment:
Lets just remove the version number from here? This will allow us to also remove the exclusion from:

https://github.com/cdk8s-team/cdk8s-core/blob/190ea12b23461aff15a46ab9db4efbe6df7f12d2/.projenrc.js#L45-L47

And then `yaml` will be upgraded automatically going forward.